contracts/BatchExchange.sol

```diff
@@ -377,6 +377,25 @@ contract BatchExchange is EpochTokenLocker {
         return IdToAddressBiMap.hasAddress(registeredTokens, addr);
     }
 
+    /** @dev View returning all byte-encoded sell orders for specified user
+      * @param user address of user whose orders are being queried
+      * @param offset uint determining the starting orderIndex
+      * @param pageSize uint determining the count of elements to be viewed
+      * @return encoded bytes representing all orders
+      */
+    function getEncodedUserOrdersPaginated(address user, uint16 offset, uint16 pageSize)
+        public
+        view
+        returns (bytes memory elements)
+    {
+        for (uint256 i = offset; i < Math.min(orders[user].length, offset + pageSize); i++) {
```

Comment:
nit: should be u16 or u32 (if we want to support overflowing)

Reply:
I'm not sure I understand this comment...